news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[Ryuusei Aika](https://osu.ppy.sh/users/7777875), known best for his extremely high BN activity and his clean mapping style, brings forth AcuticNotes' "Abst:Re:Act" in collaboration with [Reincarnation](https://osu.ppy.sh/users/4086593) (Dynamix) and [Suzuki_1112](https://osu.ppy.sh/users/3170678). The map does not let up till the ending slider art, constantly throwing crazy stream and kickslider patterns mixed with extreme spacing changes sure to challenge even the most confident of technical players. If that's too much for you, give the guest difficulty a try - it's easier than the top difficulty, but not by much!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/great-spirit.jpg)](https://osu.ppy.sh/beatmapsets/765801)
+
+[IceKalt](https://osu.ppy.sh/users/5410645)'s rendition of Armin van Buuren's "Great Spirit" captures the epic feeling of the song extremely well. Be it through expertly arranged short slider patterns or jump patterns with great usage of stop-and-go motions, each of the song's many climaxes is represented in a unique and challenging way. The calm parts are filled with beautiful and varied sliderart, making this map enjoyable all the way through!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/hotaru.jpg)](https://osu.ppy.sh/beatmapsets/580596)
+
+[Narcissu](https://osu.ppy.sh/users/1826598) manages to impress us once again with his map of Fujita Maiko's "Hotaru", bringing clean patterns as well as engaging and varying jumps that manage to perfectly capture the feeling of the song. The use of simpler rhythm lets the map focus on using it's patterns to the fullest to complement the song, supported by the well done hitsounding.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kairikou.jpg)](https://osu.ppy.sh/beatmapsets/832836)
+
+With his map of Feryquitous feat. Aitsuki Nakuru's "Kairikou", [cococolaco](https://osu.ppy.sh/users/9579526) brings us a highly polished, enjoyable map. It strikes the perfect balance between easy-going patterning and varied emphasis, making it more smooth to play than the average mapset while still representing the song in a detailed way. Dive right in and enjoy how this map and its beautiful song flow by together!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/turii.jpg)](https://osu.ppy.sh/beatmapsets/800638)
+
+With a different take on an already ranked map, [celerih](https://osu.ppy.sh/users/4696296)'s map of Zektbach's "Turii ~Panta rhei~" shows us untapped potential that the song has to offer. The map takes a focus on movement through streams, there's lots of twists in turns that aren't seen in most stream maps. All of this combined with a tricky 1/3 section and the low approach rate, the map becomes a fun challenge for the casual and experienced player. Think the Insane difficulty is too easy? Play [Muya](https://osu.ppy.sh/users/153323)'s diff for a real challenge.
```

Comment:
twists in turns -> twists and turns
The common phrase is the latter, never heard of the former before